components/02-elements/swatch/_swatch.scss

```diff
@@ -0,0 +1,86 @@
+$swatch-font-size        : $font-size-small !default;
+$swatch-margin           : 15px !default;
+$swatch-width            : 48px !default;
+$swatch-transition       : $transition-base !default;
+
+$swatch-background       : $white !default;
+$swatch-color            : $gray-light !default;
+$swatch-border-color     : $swatch-color !default;
+
+$swatch-color-hover      : $color-primary !default;
+
+$swatch-background-active: $color-primary !default;
+$swatch-border-active    : $color-primary !default;
+$swatch-color-active     : $white !default;
+
+$swatch-border           : 3px solid $swatch-border-color !default;
+$swatch-outline          : 3px solid $swatch-border-color !default;
+
+.swatch {
+    &__title {
+        font-size: $font-size-small;
+        margin-bottom: $swatch-margin;
+    }
+
+    &__handler {
+        display: flex;
+        flex-wrap: wrap;
+    }
+
+    &__option {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        min-height: $swatch-width;
+        min-width: $swatch-width;
```

Comment:
Why `min`? Can we have non-square swatches?

Reply:
Want to avoid the situation of just `width` b/c we can have more content and on the other side, we have i.e. 'S' and 'XL'. Than `min` keep all swatches in the same style. I can get rid of this idea but I think it's good coded

Reply:
👍